backend/src/bundles/hard-skills/hard-skills.model.ts

```diff
@@ -17,18 +17,18 @@ class HardSkillsModel extends AbstractModel {
     }
 
     public static override relationMappings = {
-        talents: {
-            relation: Model.ManyToManyRelation,
-            modelClass: UserDetailsModel,
-            join: {
-                from: `${DatabaseTableName.HARD_SKILLS}.${HardSkillsTableColumn.ID}`,
-                through: {
-                    from: `${DatabaseTableName.TALENT_HARD_SKILLS}.${TalentHardSkillsTableColumn.HARD_SKILL_ID}`,
-                    to: `${DatabaseTableName.TALENT_HARD_SKILLS}.${TalentHardSkillsTableColumn.USER_DETAILS_ID}`,
-                },
-                to: `${DatabaseTableName.USER_DETAILS}.${UserDetailsTableColumn.ID}`,
-            },
-        },
+        // talents: {
+        //     relation: Model.ManyToManyRelation,
+        //     modelClass: UserDetailsModel,
+        //     join: {
+        //         from: `${DatabaseTableName.HARD_SKILLS}.${HardSkillsTableColumn.ID}`,
+        //         through: {
+        //             from: `${DatabaseTableName.TALENT_HARD_SKILLS}.${TalentHardSkillsTableColumn.HARD_SKILL_ID}`,
+        //             to: `${DatabaseTableName.TALENT_HARD_SKILLS}.${TalentHardSkillsTableColumn.USER_DETAILS_ID}`,
+        //         },
+        //         to: `${DatabaseTableName.USER_DETAILS}.${UserDetailsTableColumn.ID}`,
+        //     },
+        // },
```

Comment:
Why did you remove relation mappings ?

Reply:
Backend doesn't work with this code, but I think it's necessary part for the DB logic
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/42421305/15d6920e-e403-4f1c-9dc5-6d79ece225a2)

Reply:
Got it, try change this logic to: 
modelClass: join(__dirname, pathToModel),

Reply:
Temporary fixed. 
```modelClass: "",```